tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,33 @@
+import helpscoutBeacon from 'codecrafters-frontend/tests/pages/components/helpscout-beacon';
+
+module('Acceptance | helpscout-beacon', function (hooks) {
+  setupApplicationTest(hooks);
+  setupAnimationTest(hooks);
+
+  test('beacon renders on catalog page', async function (assert) {
+    testScenario(this.server);
+    await visit('/catalog');
+    await animationsSettled();
+
+    await this.pauseTest();
```

ESLint

[error] Do not commit `pauseTest()`

(ember/no-pause-test)